android/app/src/main/java/org/fossasia/openevent/fragments/FeedFragment.java

```diff
@@ -80,9 +87,12 @@ private void downloadFeed() {
                             .getString(R.string.refresh_failed), Snackbar.LENGTH_LONG)
                             .setAction(R.string.retry_download, view -> refresh()).show();
                     Timber.d("Refresh not done");
+                    showProgressBar(false);
                 }, () -> {
+                    downloadProgressDialog.setProgress(100);
```

Comment:
Are you incrementing the progress or it just turns from 0 to 100. If latter, then intermediate progress bar is preferred